code/go/0chain.net/smartcontract/storagesc/alloation2_test.go

```diff
@@ -787,6 +790,23 @@ func testNewAllocation(t *testing.T, request newAllocationRequest, blobbers Sort
 		ToClientID:   storageScId,
 		CreationDate: creationDate,
 	}
+	access := config.DbAccess{
```

Comment:
why do we need eventdb connection in testNewAllocation?

Reply:
unit test of edb should in dbs/event package only

Reply:
since we have add the `addTransfer` and `addMint` function in the `alloation2_test.go`, so should I move this test to dbs/event only?